tests/vmi_lifecycle_test.go

```diff
@@ -93,6 +93,29 @@ var _ = Describe("[rfe_id:273][crit:high][vendor:cnv-qe@redhat.com][level:compon
 		tests.EnsureKVMPresent()
 	})
 
+	Context("when virt-launcher is deleted", func() {
+		It("should label the node with kubevirt.io/schedulable=false", func() {
```

Comment:
We have also a test which looks like this:

```go
			It("[test_id:1634]the node controller should react", func() {

				// Update virt-handler heartbeat, to trigger a timeout
				data := []byte(fmt.Sprintf(`{"metadata": { "annotations": {"%s": "%s"}}}`, v1.VirtHandlerHeartbeat, nowAsJSONWithOffset(-10*time.Minute)))
				_, err = virtClient.CoreV1().Nodes().Patch(nodeName, types.StrategicMergePatchType, data)
				Expect(err).ToNot(HaveOccurred(), "Should patch node successfully")
```

I think that with your code-change it would not properly test this section anymore:


```go
				// it will take at least 45 seconds until the vmi is gone, check the schedulable state in the meantime
				By("marking the node as not schedulable")
				Eventually(func() string {
					node, err := virtClient.CoreV1().Nodes().Get(nodeName, metav1.GetOptions{})
					Expect(err).ToNot(HaveOccurred(), "Should get node successfully")
					return node.Labels[v1.NodeSchedulable]
				}, 20*time.Second, 1*time.Second).Should(Equal("false"), "The node should not be schedulable")

```

I think that we need to tweak the patch in this test there, to explicitliy set the node to schedulable too, since now the eviction of virt-handler would already set it to false, while we want to test here the node-controller which should do that if virt-handler couldn't.

Reply:
ahh yes, you are correct. Thank you catching that. I have updated it as you suggested to make sure this is testing node-controller and not the virt-handler ds